ui/learn/src/run/runCtrl.js

```diff
@@ -47,11 +47,15 @@ module.exports = function (opts, trans) {
   opts.route = 'run';
   opts.stageId = stage.id;
 
+  const isRestarting = lichess.tempStorage.makeBoolean('learn.restarting');
```

Comment:
could it not just be a `let` or `m.prop`?

Reply:
I'm not familiar with mithril, so maybe there's a better way but since it reloads the route, I don't think a simple `let` or prop works since it completely reruns the function and recreates everything as far as I can tell.